libcontainer/cgroups/devices/devicefilter_test.go

```diff
@@ -120,14 +120,21 @@ block-8:
         51: MovImm32 dst: r0 imm: 1
         52: Exit
 block-9:
-// /dev/pts (c, 136, wildcard, rwm, true)
+// tuntap (c, 10, 200, rwm, true)
```

Comment:
Could you add a comment line to describe the background of this revert?

Reply:
You mean here, in the test file?

I don't think it's needed here in the test -- in here we just check that the eBPF generated is as expected, and since we've re-added tuntap, we have to modify the expected program.

Reply:
Ah, it should be in `spec_linux.go`

Reply:
Added; PTAL